packages/snap/src/zkCertHandler.ts

```diff
@@ -52,7 +54,7 @@ export function getZkCertStorageOverview(zkCertStorage: ZkCertRegistered[]) {
       expirationDate: zkCert.expirationDate,
     };
     if (zkCert.zkCertStandard === KnownZkCertStandard.ZkKYC) {
```

Comment:
This is better expressed as a [type predicate](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates).